neonvm/apis/neonvm/v1/virtualmachine_types.go

```diff
@@ -124,6 +137,14 @@ type VirtualMachineSpec struct {
 	EnableSSH *bool `json:"enableSSH,omitempty"`
 }
 
+func (spec *VirtualMachineSpec) Resources() *VirtualMachineResources {
```

Comment:
out of curiosity: why return a pointer instead of the object itself?

Reply:
I usually write code in a way where all manipulation with structs is done through pointers. This is to not think whether struct is small enough to copy it around without performance impact. Or maybe this is java experience talking deep inside me.

I've changed it here, maybe I'll get used to it eventually.

Reply:
Haha, fair enough, makes sense :)